tests/clients.py

```diff
@@ -0,0 +1,21 @@
+from algosdk.v2client.algod import AlgodClient
+
+# from algosdk.kmd import KMDClient
+# from algosdk.v2client.indexer import IndexerClient
+
+DEVNET_TOKEN = "a" * 64
+ALGOD_PORT = 4001
+# KMD_PORT = 4002
+# INDEXER_PORT = 8980
+
+
+def get_algod() -> AlgodClient:
+    return AlgodClient(DEVNET_TOKEN, f"http://localhost:{ALGOD_PORT}")
+
+
+# def get_kmd() -> KMDClient:
```

Comment:
@tzaffi Minor:  Friendly pointer to commented out code.  Can it be removed?

Reply:
ok... the reason I was keeping it around is too speculative at this point